manifests/init.pp

```diff
@@ -75,6 +79,33 @@
     path => $certs::iop_advisor_engine::client_ca_cert,
   }
 
+  podman::quadlet { $service_name:
+    ensure       => 'present',
+    quadlet_type => 'container',
+    user         => 'root',
+    defaults     => {},
+    settings     => {
+      'Unit'      => {
+        'Description' => 'Advisor Engine',
+      },
+      'Container' => {
+        'Image'   => $image,
+        'Network' => 'host',
+        'Volume'  => [
+          "${log_dir}/uploads:/opt/app-root/src/uploads:z",
+          "${log_dir}/failed:/opt/app-root/src/failed:z",
+          "${log_dir}/logs:/opt/app-root/src/logs:z",
```

Comment:
Using `:z` here will break if people run with selinux disabled. (We don't officially support that, but people do)